Localization/ensemble_kalman_filter/ensemble_kalman_filter.py

```diff
@@ -167,7 +167,7 @@ def plot_covariance_ellipse(xEst, PEst):  # pragma: no cover
 
     x = [a * math.cos(it) for it in t]
     y = [b * math.sin(it) for it in t]
-    angle = math.atan2(eig_vec[big_ind, 1], eig_vec[big_ind, 0])
+    angle = math.atan2(eigvec[1, bigind], eigvec[0, bigind])
```

Comment:
typo? eigvec -> eig_vec, bigind -> big_ind?

Reply:
Sorry, those were typos. I fixed it in [a0d1c74](https://github.com/AtsushiSakai/PythonRobotics/pull/407/commits/a0d1c746fa6b44d5deb493757320f811b85e1780).